docs/slack-connector/installation.md

```diff
@@ -1,13 +1,63 @@
+- [Installation in Kyma as an Add-On](#installation-in-kyma-as-an-add-on)
   - [Prerequisites](#prerequisites)
   - [Steps](#steps)
+  - [Verification](#verification)
+  - [Removal](#removal)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites-1)
+  - [Steps](#steps-1)
 
 ## Overview
 
-The Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
+The Slack Connector is a component which allows interaction with Slack API from inside of Kyma environment. The simplest way to install Slack Connector in Kyma is to install it as an Add-On.
+
+## Installation in Kyma as an Add-On
+
+### Prerequisites
+
+- Slack Bot with desired privileges installed to destination workspace. Tutorial provided by Slack on how to setup an application can be found [here](https://api.slack.com/bot-users#getting-started).
+- Access to Kyma Console.
+
+> **OPTIONAL:** Install default application following these steps, but pay attention to the fact that it has **full permissions** in workspace.
+>
+> 1. Go to the [authentication page](https://auth-slack.herokuapp.com/). Click **Add to Slack** button which redirects you to another page. Select your desired workspace and click **Allow**.
+>       - **NOTE:** If the link does not work see [this tutorial](https://api.slack.com/docs/oauth#flow) in the Slack API documentation to create your own application.
+> 2. Copy the Bot Authentication Token and/or Slack Signing Secret. You will need it later in the installation process.
+
+### Steps
+
+1. In Kyma console access the **Add-Ons Config** menu.
+2. Click **Add New Configuration** and in the **Urls\*** field paste this URL:
+
+   ```http
+   github.com/kyma-incubator/hack-showcase//addons
+   ```
+
+3. Go to namespace in which you want to install the Connector.
+4. Find the Add-On in Service Catalog and click it.
+5. Click **Add** and select the installation plan. Fill in all required fields and click **Create Instance**.
+6. Go to **Services** tab of Service Catalog. After provisioning and automatic registration of application's resources the Service Class of Slack Connector can be found here.
+7. Click it to enter its specification screen and click **Add once** and then **Create Instance**.
+
+After the service is created it can be easily bound to Lambda Function to allow use of Slack Events and Web API.
+
+### Verification
+
+To verify if everything is configured correctly check if Add-Ons and Service instances in **Instances** area of Service Catalog have status <span style="color:green">*RUNNING*</span>.
+
+### Removal
+
+Basically, to correctly remove all resources of Slack Connector you must delete them in order reverse to installation steps.
+> **NOTE:** Wait until deprovisioning and removing of all elements is complete before proceeding to next step, because e.g. after removing ServiceClass removal of ServiceInstance is impossible.
```

Comment:
```suggestion
> **NOTE:** Wait until deprovisioning and removing of all elements is complete before proceeding to next step to avoid possible errors. For example, after removing ServiceClass, the removal of ServiceInstance is impossible.
```

- I'd change it from **NOTE** to  **CAUTION** as it's really important and omission will result in an error. 
- Don't use `e.g.`, use `For example`; also, don't use it mid-sentence.